src/test/java/hudson/plugins/jira/JiraCreateReleaseNotesTest.java

```diff
@@ -65,9 +65,8 @@ public class JiraCreateReleaseNotesTest {
 
     @Before
     public void createCommonMocks() throws IOException, InterruptedException {
-        when(build.getProject()).thenReturn(project);
```

Comment:
Fixing a Mockito warning about an unnecessary stub by removing the unnecessary stub.